crates/gpui/src/text_system.rs

```diff
@@ -78,18 +78,15 @@ impl TextSystem {
 
     /// Get a list of all available font names from the operating system.
     pub fn all_font_names(&self) -> Vec<String> {
-        let mut names: BTreeSet<_> = self
-            .platform_text_system
-            .all_font_names()
-            .into_iter()
-            .collect();
-        names.extend(self.platform_text_system.all_font_families());
```

Comment:
Therefore, merging the results of `all_font_names` with `all_font_families` is unnecessary since the former always includes the latter. As a result, I removed the `all_font_families` function.